openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -167,6 +167,8 @@ <h2 class="staff-info__title ui-staff__title">
         </div>
     </div>
 
+    <!-- Conditional not really required because button will be hidden. -->
```

Comment:
It definitely works without it, in that the HTML is hidden. However, it feels odd to include and hide HTML that isn't relevant, so I was wondering if it was better to not include the HTML at all if staff assessment is not required.